tests/microgrid/mock_api.py

```diff
@@ -100,6 +102,17 @@ def add_component(
                     inverter=InverterMetadata(type=inverter_type),
                 )
             )
+        elif (
+            component_category == ComponentCategory.COMPONENT_CATEGORY_GRID
+            and max_current is not None
```

Comment:
if `max_current` should be always different than zero then I'd suggest to use `and max_current` instead as this covers the check for not `None` and not `0`

Reply:
> if `max_current` should be always different than zero

I did not make that assumption here. I think a mock should not have conditions on data values. The functional implementation should have it instead.

But it raises a good point. In this case, would it make sense to raise an error in the `Grid` module if `fuse.max_current == 0`, given the low-level layer it exists in? Or should it be handled by the power distributor?

Reply:
> I did not make that assumption here. I think a mock should not have conditions on data values. The functional implementation should have it instead.

You're right, agreed.

> would it make sense to raise an error in the Grid module if fuse.max_current == 0, given the low-level layer it exists in? Or should it be handled by the power distributor?

I think It does make sense to do it in the Grid module to prevent programming or setups errors. And additionally can be done in the power distributor. Maybe @matthias-wende-frequenz has another opinion on this matter?

Reply:
Okay, then I think we can resolve this discussion here, and pass this on to the power distributor.